api/utils/utils.go

```diff
@@ -254,15 +263,38 @@ func DecryptText(encryptedText string) (string, error) {
 	return string(decryptedText), nil
 }
 
-func LoadEnvVars() map[string]interface{} {
-	return map[string]interface{}{
-		"DATABASE_URL":     os.Getenv("DATABASE_URL"),
-		"REDIS_HOST":       os.Getenv("REDIS_HOST"),
-		"REDIS_PORT":       os.Getenv("REDIS_PORT"),
-		"APP_PORT":         os.Getenv("APP_PORT"),
-		"TEST_MODE":        os.Getenv("TEST_MODE"),
-		"ENCRYPT_PASSWORD": os.Getenv("ENCRYPT_PASSWORD"),
+func LoadEnvVars() (map[string]interface{}, error) {
+	databaseURL := os.Getenv("DATABASE_URL")
+	redisHost := os.Getenv("REDIS_HOST")
```

Comment:
It could be nice to have default values for `REDIS_HOST` and `REDIS_PORT` as `localhost` and `5432` respectively.